### PR TITLE
IAT-3599: Some data written incorrectly to Tutorial SAAIF files

### PR DESCRIPTION
**Pull request by gregcharles**
Note: I'm seeing a failure in QtiAssessmentItemTranslationServiceIT, but it doesn't appear related to these changes.

### CODE REVIEW

**Comment by gregcharles** [comment]
> I approve but would like @agorina or @wtritch to weigh in on the QTI test failure.

I'm seeing:
`<qti-custom-interaction response-identifier="RESPONSE" data-interaction-identifier="sbac:grid-item">`
vs.
`<qti-custom-interaction response-identifier="RESPONSE" data-interaction-identifier="sbac:equation">`
          
Also, big differences to the `<template>` entity under `<qti-custom-interaction>`

And:
`<qti-custom-operator class="qti.sbac.customOperators.GRAPHIC.PREPROCESSRESPONSE" definition="response=RESPONSE"/>`
vs.
`<qti-custom-operator class="qti.sbac.customOperators.EQ.PREPROCESSRESPONSE" definition="response=RESPONSE"/>`

File: common/src/main/java/org/opentestsystem/ap/common/saaif/transformer/SaaifTransformer.java [hunk 1]
```diff
@@ -349,7 +354,7 @@ public SmarterAppMetadata mapToMetadata(Item item, List<String> enemyItems) {
         }
 
         metadata.getStandardPublication().clear();
-        if (!TYPE_STIM.equalsIgnoreCase(item.getType()) && !TYPE_TUT.equalsIgnoreCase(item.getType())) {
```
Comment (alex-ponce) [inline common/src/main/java/org/opentestsystem/ap/common/saaif/transformer/SaaifTransformer.java]: Does this imply that the `isStimulus` and `isTutorial` item properties are incorrect?

Comment (gregcharles) [reply common/src/main/java/org/opentestsystem/ap/common/saaif/transformer/SaaifTransformer.java]: Hmm, I don't think so. 

> Does this imply that the `isStimulus` and `isTutorial` item properties are incorrect?

I'm not sure I understand the question. I changed the way it's checking for the item types to what seemed to be newer methods that just hadn't been retrofitted everywhere yet. I did dig into the code and was confident that the item.is<Type>() methods were equivalent to <Type String>.equals(item.getType()). Is that your concern, or am I missing something?

Comment (millerts) [reply common/src/main/java/org/opentestsystem/ap/common/saaif/transformer/SaaifTransformer.java]: I agree with @gregcharles.  I think either are acceptable.

**Comment by gregcharles** [comment]
Note: fixed Qti test failure in another branch. It is unrelated to these changes.

**Pull request merged**